OrbitVulkanLayer/Main.cpp

```diff
@@ -0,0 +1,270 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
```

Comment:
This file obviously doesn't contain a `main` function as this is a shared library, but that's what one would expect from a file named `Main.cpp`. I don't really have an idea but could we find a better name for the file?

Reply:
In the compiler/static analysis community, those are typically called "entry points". So Maybe `VulkanLayerEntryPoints`? What do you think?

Reply:
It sounds good, it matches what they actually are. It could also be just `EntryPoints`.

Reply:
Went for EntryPoints